src/pages/get-eth.tsx

```diff
@@ -182,309 +150,243 @@ const GetEthPage = ({
-          </InlineLink>
-        </StyledCard>
-        <StyledCard
-          emoji=":building_construction:"
-          title={t("page-get-eth-earn")}
-          description={t("page-get-eth-earn-desc")}
-        >
-          <InlineLink href="/dao/">
-            {t("page-get-eth-daos-link-desc")}
-          </InlineLink>
-        </StyledCard>
-        <StyledCard
-          emoji=":busts_in_silhouette:"
-          title={t("page-get-eth-peers")}
-          description={t("page-get-eth-peers-desc")}
-        >
-          <InlineLink href="/wallets/">
-            {t("page-get-eth-wallets-link")}
-          </InlineLink>
-        </StyledCard>
-        <StyledCard
-          emoji=":robot:"
-          title={t("page-get-eth-dex")}
-          description={<Translation id="page-get-eth:page-get-eth-dex-desc" />}
-        >
-          <InlineLink href="#dex">{t("page-get-eth-try-dex")}</InlineLink>
-        </StyledCard>
-        <StyledCard
-          emoji=":key:"
-          title={t("page-get-eth-wallets")}
-          description={t("page-get-eth-wallets-purchasing")}
-        >
-          <InlineLink href="/wallets/">
-            {t("page-get-eth-wallets-link")}
-          </InlineLink>
-        </StyledCard>
-        <StyledCard
-          emoji=":shield:"
-          title={t("page-get-eth-staking")}
-          description={t("page-get-eth-staking-desc")}
-        >
-          <InlineLink href="/staking">
-            {t("page-get-eth-staking-link-desc")}
-          </InlineLink>
-        </StyledCard>
-        <Content>
-          <Text>
-            <Text as="em">
+            <InlineLink href="#dex">{t("page-get-eth-try-dex")}</InlineLink>
+          </StyledCard>
+          <StyledCard
+            emoji=":key:"
+            title={t("page-get-eth-wallets")}
+            description={t("page-get-eth-wallets-purchasing")}
+          >
+            <InlineLink href="/wallets/">
+              {t("page-get-eth-wallets-link")}
+            </InlineLink>
+          </StyledCard>
+          <StyledCard
+            emoji=":shield:"
+            title={t("page-get-eth-staking")}
+            description={t("page-get-eth-staking-desc")}
+          >
+            <InlineLink href="/staking">
+              {t("page-get-eth-staking-link-desc")}
+            </InlineLink>
+          </StyledCard>
+        </div>
+
+        <div className="mb-8">
+          <p>
+            <em>
               {t("common:listing-policy-disclaimer")}{" "}
               <InlineLink href="https://github.com/ethereum/ethereum-org-website/issues/new/choose">
                 {t("listing-policy-raise-issue-link")}
               </InlineLink>
-            </Text>
-          </Text>
+            </em>
+          </p>
           <InfoBanner emoji=":wave:" shouldCenter mt={8}>
             {t("page-get-eth-new-to-eth")}{" "}
             <InlineLink href="/eth/">
               {t("page-get-eth-whats-eth-link")}
             </InlineLink>
           </InfoBanner>
```

Comment:
<img width="721" alt="image" src="https://github.com/user-attachments/assets/005b9d69-4f5d-4ce8-bfed-abe7b6fd00fb">

Related to the `InfoBanner` component, noting our contrast ratio has taken a hit with links on the default banner variant background.

Not a blocker for this PR

Reply:
Have some work done locally trying to update the InfoBanners... will try to get a PR up within the next week

Reply:
The idea we had with these InfoBanner's was to replace them with [the new Alerts](https://dev--63b7ea99632763723c7f4d6b.chromatic.com/?path=/story/molecules-action-feedback-alerts--variants). Will do that in the scope of this PR but we could replace all of the InfoBanners perhaps.

Reply:
> Will do that in the scope of this PR but we could replace all of the InfoBanners perhaps.

@pettinarip _In_ the scope of this PR? There are over 500 instances of this component, touching almost 500 files... 😆 I would strongly advise we not make that part of this PR

Reply:
hah I see your Discord response: out of scope